common/src/main/java/com/microsoft/identity/common/internal/cache/ADALOAuth2TokenCache.java

```diff
@@ -75,7 +75,7 @@
     static final String ERR_UNSUPPORTED_OPERATION = "This method is unsupported.";
 
     private static final String TAG = ADALOAuth2TokenCache.class.getSimpleName();
-    private static final String SHARED_PREFERENCES_FILENAME = "com.microsoft.aad.adal.cache";
+    public static final String SHARED_PREFERENCES_FILENAME = "com.microsoft.aad.adal.cache";
```

Comment:
Can we expose a getter for this value instead and drop a message into the logs about reading it?

Reply:
22f1cefdb1c00e4110d1ce2e318be6c823fc4cbb